.github/actions/spell-check/expect.txt

```diff
@@ -2292,59 +1824,46 @@ wtsapi
 WTSAT
 Wubi
 WVC
-Wwan
 Wwanpp
-xamlstyler
-Xavalon
+WWL
```

Comment:
This is the name of a team, I think. Why are we removing from names.txt and adding in expect.txt?